blocks/library/latest-posts/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * WordPress dependencies
+ */
+import Placeholder from 'components/placeholder';
```

Comment:
I think we should use `import { Placeholder } from 'components';` to avoid code duplication when bundling the modules.